.changeset/forty-toys-dress.md

```diff
@@ -0,0 +1,5 @@
+---
+"@aptos-labs/wallet-adapter-core": patch
+---
+
+Support Lodable wallet
```

Comment:
could you also add the implementation for signAndVerify change?

Reply:
updated